.github/workflows/test.yml

```diff
@@ -8,6 +8,24 @@ on:
   workflow_call:  # Allow for the publish workflows to run the tests by calling this workflow
 
 jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up Python 3.8
```

Comment:
considering 3.8 will stop being support in less than a month, it is best you are more opinonated with this ;)

Reply:
Done ✅